amqp/enabled_plugins

```diff
@@ -0,0 +1 @@
+[rabbitmq_auth_mechanism_ssl,rabbitmq_management].
```

Comment:
The trailing dot looks weird. Intended/necessary, or a typo?

Reply:
necessary (https://www.rabbitmq.com/plugins.html#ways-to-enable-plugins); I agree it looks weird